scout/server/blueprints/cases/controllers.py

```diff
@@ -841,8 +820,13 @@ def update_individuals(store, institute_obj, case_obj, user_obj, ind, age, tissu
 
     case_obj["individuals"] = case_individuals
 
-    verb = "update_individual"
-    _update_case(store, case_obj, user_obj, institute_obj, verb)
```

Comment:
Changed to not pass the verb, but instead use two different EventHandler functions. Also create link externally, like for most events.